khtools/compare_kmer_content.py

```diff
@@ -62,18 +65,29 @@
                                   'NA': 0})
 divergence_estimates = divergence_estimates.sort_values()
 
-
 KSIZES = 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, \
     21, 23, 24, 25
 COLUMNS = 'id1', 'id2', 'ksize', 'jaccard'
 
+
 # Hydrophobic/hydrophilic mapping
 
 # From: Brüne, D., Andrade-Navarro, M. A., & Mier, P. (2018).
 # Proteome-wide comparison between the amino acid composition of domains and
 # linkers. BMC Research Notes, 1–6. http://doi.org/10.1186/s13104-018-3221-0
 
 
+def sanitize_id(value):
+    """
+    Normalizes string, converts to lowercase, removes non-alpha characters,
```

Comment:
I don't think the docstring is right.

the code you used just replaces the | with __